src/main/java/se/sundsvall/supportmanagement/api/validation/impl/ValidStatusTagConstraintValidator.java

```diff
@@ -1,21 +1,31 @@
 package se.sundsvall.supportmanagement.api.validation.impl;
 
+import static java.util.Collections.emptyList;
+import static java.util.Optional.ofNullable;
+
+import java.util.List;
+
 import javax.validation.ConstraintValidator;
 import javax.validation.ConstraintValidatorContext;
 
 import org.springframework.beans.factory.annotation.Autowired;
 
+import se.sundsvall.supportmanagement.api.model.metadata.Status;
 import se.sundsvall.supportmanagement.api.validation.ValidStatusTag;
-import se.sundsvall.supportmanagement.service.TagService;
+import se.sundsvall.supportmanagement.service.MetadataService;
 
 public class ValidStatusTagConstraintValidator extends AbstractTagConstraintValidator implements ConstraintValidator<ValidStatusTag, String> {
```

Comment:
Remove tag-suffix here as well (both in the name of this class and the parent class)?

Reply:
I'll leave it to Micke to change name as I will f*ck it up for him if I do the refactoring in my branch when he refactors the validators in his... I'll be sure to tell him in his PR if you don't 🙂!